packages/netlify-cms-lib-widgets/src/stringTemplate.ts

```diff
@@ -115,7 +115,7 @@ export function compileStringTemplate(
   date: Date | undefined | null,
   identifier = '',
   data = Map<string, unknown>(),
-  processor?: (value: string) => string,
+  processor?: (value: string, key: string) => string,
```

Comment:
Ideally it should be `processor?: (key: string, value: string) => string,` But I figured it was a less of a sweeping change to append the parameter instead.